system/cfme-setup.sh

```diff
@@ -40,9 +40,6 @@ EOF
 [ -x /sbin/restorecon ] && /sbin/restorecon -R -v /var/www/miq/vmdb/certs
 [ -x /sbin/restorecon ] && /sbin/restorecon -R -v /var/www/miq/system/logrotate_free_space_check.sh
 
-# relabel the pg_log directory in postgresql datadir, but defer restorecon
-# until after the database is initialized during firstboot configuration
-/usr/sbin/semanage fcontext -a -t var_log_t "/opt/rh/postgresql92/root/var/lib/pgsql/data/pg_log(/.*)?"
```

Comment:
This is 2 PRs around 1 central theme.

1. When you mount `DATABASE_DISK_MOUNT_POINT` setup all of it to have the proper privileges.
2. Just use the default fcontext for all the directories in `DATABASE_DISK_MOUNT_POINT`

It is possible that task 2 is wrong. If that is the case I'll remove the change to `cfme-setup.sh`.